### ISSUE TITLE
ONGOING: Manage Telegram growth, focus, support

### ISSUE THREAD

**Issue by kitblake** [rchain/bounties#735]
_Describe how the building and implementation of this task will benefit RChain:_

The organic growth of the Telegram groups continues. Admins are keeping the conversation focused on the tech as opposed to the market. Lots of the same questions get asked and with the answers admins often send users on to Github, Jira, and the Dev site. A lot of Members onboard via Telegram. Admins steer users to join and provide support for the process. 

**Current groups**
https://t.me/rchain_coop
https://t.me/rchain_official (announcement channel)
https://t.me/rchain_africa (#592)
https://t.me/rchain_cn
https://t.me/rchain_fr
https://t.me/rchain_nl
https://t.me/rchain_ru (#596)

_Estimated Budget of Task:_ **$2480**
_Estimated Timeline Required to Complete the Task:_ **the current month**
_Measure of Completion:_ **see Combot statistics for the larger groups**

--
***Legal***
*Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.*

**Comment by kitblake** [comment]
This is the May activity for the RChain_Coop group:
https://combot.org/chat/-1001123521235?start=01%2F05%2F2018&end=31%2F05%2F2018

**Comment by kitblake** [comment]
We're testing the DaySandBox bot, which:
"Removes forwarded and link-containing posts from new group users (joined less than [24] hours ago)."
It's configurable.

**Comment by kitblake** [comment]
The DaySandBox bot has been added to @rchain_coop with default settings.

Common commands:
/daysandbox_get publog
/daysandbox_set publog=yes
/daysandbox_get safe_hours
/daysandbox_set safe_hours=24

**Comment by Preshcode** [comment]
I think you're doing a great job with management of the telegram group.

**Comment by kitblake** [comment]
May activity for the RChain_Africa group:
https://combot.org/chat/-1001196019499?start=01%2F05%2F2018&end=31%2F05%2F2018

**Comment by jasoncruzzy** [comment]
@kitblake  The telegram  groups has made tremendous growth and it has provided both existing and new members with relevant information on how to grow the coop . RChain Africa group will  continually intensify its efforts to involve more RAM

**Comment by kitblake** [comment]
Yesterday we had somebody PMing users with spam, including @ddayan who reported it in the group. Then the spammer left and returned under a new name with @ddayan's avatar. So If you're dubious about a user, inspect the avatar. If the old avatar is obviously a different person, ban.

**Comment by kitblake** [comment]
The main group just passed 2500 members.

**Comment by kitblake** [comment]
The @daysandbox_bot is proving really useful. If not already done, admins should consider installing it. 

Some numbers at the end of the month:
@rchain_coop 2624 members
@rchain_africa 51
@rchain_china 242
@rchain_france 24
@rchain_nl 51
@rchain_ru 7
QQ group China: 893

**Comment by pmoorman** [comment]
@kitblake can you post a status update again (and lead a suggestion on budget & rewards distribution on this issue)?

**Comment by kitblake** [comment]
Numbers at the end of the month:
@rchain_coop 2698 members
@rchain_africa 59
@rchain_china 244
@rchain_france 24
@rchain_nl 51
@rchain_ru 9
QQ group China: 908

Member growth is clearly slowing, not sure why. Summertime? Activity has been consistent with previous months but shows a small drop in July. 

Accordingly I've dropped the budget a bit in my vote, to 2240, and I used combat statistics to distribute the rewards.

**Comment by Ojimadu** [comment]
Yeah, July was slow except for this past week where talks about marketing and the Validator discussions has spurred interest. August would be better I believe.

**Comment by AyAyRon-P** [comment]
Hi @kitblake 

Is this still ongoing? Are there any updates? Do we need to vote on this for the month of August? 

If so please place the ONGOING: tag to the front of the project.

**Comment by kitblake** [comment]
@AyAyRon-P Yes, yes, yes, and done (added the missing ONGOING to the title). And thanks.

Numbers at the end of the month:
@rchain_coop 2828 members
@rchain_africa 69
@rchain_china 262
@rchain_france 24
@rchain_nl 54
@rchain_ru 8
QQ group China: 947

Rising less rapidly, but steadily. In the last days activity has been ratcheting up with the Validator sale and RCon3 approaching. Making the budget a bit more than last month, 2320. 

@Ojimadu gets the crown again (but he'll be offline for the next weeks, so other admins need to pick up the slack).

**Comment by dckc** [comment]
rewarded in 201805 thru 201808

**Comment by Ojimadu** [comment]
@dckc @kitblake @patrickM727 @ddayan @ysgjay @pmoorman

From the meeting notes of the last impromptu TAC meeting, I see that there is a suggestion about not having to pay for "monitoring social media". I wasn't at the meeting so am airing my opinion here (as I have also done on the document). I am not of the opinion that we shouldn't reward admin and management of social channels.
There's a lot more into managing these channels than "just monitoring and deleting spam messages". RChain has so many moving part and keeping up with different chat channels, reading documents, watching videos and generally trying to keep up with the community takes significant time and dedication that's where much of the overhead comes from.
Also, there's another issue of responding to questions, engaging with the community members and managing general perception this is another aspect on its own. In crypto Telegram, reddit, Twitter and sometimes bitcointalk.org are channels that the crypto-community are very active on and they need to be actively managed as activities on these channels generally inform the perception outsiders perception of our community. At the last debrief (debrief 97) it was clearly pointed out that our social media outreach is not meeting up

**Comment by pmoorman** [comment]
I think @kitblake is in the best position to answer this, since he had the most outspoken opinion on it (that is: everyone else just went along with what he suggested).

To me, what you're saying makes perfect sense. I think Kit's suggested that most volunteer management would get done anyway, independent of whether it's paid or not. 

But really, it's best to discuss with Kit about it.

**Comment by dckc** [comment]
I don't see anyone disgreeing with you that the monitoring should be done, @ojimadu.

But I don't see anything in your argument to counter the notion that we can get it done on a volunteer basis.

**Comment by Ojimadu** [comment]
@dckc, I believe it can get it done on a volunteer basis but IMO, that's somewhat like saying, since RChain is open source we should just have volunteers build it out and not pay anyone for it. 
Rewarding people for their time and effort whether volunteering or not gives a sense of responsibility and motivation and it also affect the amount of time people are willing to put into a given task, the scale of preference changes and this would reduce the overall effectiveness and this isn't needed especially when now. 
Also, it norm in most Crypto communities to have a mix volunteer and paid admin but having everything up for 'volunteering' is a risk that's not worth taking.

FWIW, I've been helping out on this too and I have been the most active admin on Telegram and I am not of the opinion of leaving it as "volunteer only".

**Comment by kitblake** [comment]
What timing, the @rchain_coop Telegram channel just hit 3000 members. 🎉

**Comment by kitblake** [comment]
@dckc We're working on the sept rewards. It wasn't my intention to 'stiff' the diligent admins.

**Comment by kitblake** [comment]
@Ojimadu I'm wearing two hats on this one. I completely agree with you, about the need in general and the value of (especially your) engagement.

But my manager persona says we can do it cheaper. Maybe we don't have to pay at all because, if you don't engage, other people may step up. For free.

If I'm proven wrong I'll actually be happy about it. Then we'll need to marshall some budget. But we need proof.

**Comment by Ojimadu** [comment]
Perhaps, we can try but my opinion hasn't changed.
Cheers :)
